### ISSUE TITLE
Implement GUI for use cases in the vehicle properties context area 

### ISSUE THREAD

**Issue by stefanieschreiner** [i315315/amos2022ws02-automotive-test-app#29]
## User story
1. As a User
2. I want to see a list of possible use cases in the context of vehicle properties 
3. So that it is possible to test vehicle properties use cases

## Acceptance criteria
* In vehicle properties context there is a list of all available use cases (currently Night Mode, Remaining Range) 
* The list should be easily extendable.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
